tests/common.go

```diff
@@ -4353,6 +4364,83 @@ func CreateS3BackupLocationWithContext(name string, uid, cloudCred string, cloud
 	return nil
 }
 
+// CreateS3BackupLocationWithSseType creates backup location for S3 with SSE type
+func CreateS3BackupLocationWithSseType(name string, uid, cloudCred string, cloudCredUID string, bucketName string, orgID string, encryptionKey string, sseS3EncryptionType api.S3Config_Sse) error {
```

Comment:
Why do we need a separate function for SSE Type. Can we not just add ` sseS3EncryptionType api.S3Config_Sse` as an optional argument in `CreateS3BackupLocation` itself?

Reply:
In that case all test needs to fetch sseS3EncryptionType and pass to CreateS3BackupLocation in case of global run

Reply:
No. The `sseS3EncryptionType` will be optional. Only SSE specific tests will provide it. For others the function signature will not change

Reply:
yes correct in case of specific case and ideally we should follow that, but in this case of global run we are reading encryption type from the env variable , tests need to read from the env and pass to createS3Backup location 
we can discuss on this

Reply:
We can read the argument or env variable both in function right ? If argument not passed, we can check env variable.